src/providers/wms/qgswmsprovider.cpp

```diff
@@ -1149,10 +1157,10 @@ void QgsWmsProvider::addWmstParameters( QUrlQuery &query )
   }
 
   // If the data provider has bi-temporal properties and they are enabled
-  if ( uri.hasParam( QStringLiteral( "reference_time" ) ) &&
-       !uri.param( QStringLiteral( "reference_time" ) ).isEmpty() )
+  if ( uri.contains( QStringLiteral( "reference_time" ) ) &&
+       !uri.value( QStringLiteral( "reference_time" ) ).toString().isEmpty() )
   {
-    QString time = uri.param( QStringLiteral( "reference_time" ) );
+    QString time = uri.value( QStringLiteral( "reference_time" ) ).toString();
```

Comment:
Is there  a reson why "reference_time" contains an underscore and the other parameters are camel case?

Reply:
There isn't, I think I took that from WMS-T spec, will update it.

Reply:
Yes, better, thank you: just to be consistent.